assets/models/system/tiiuae-falcon-40b-instruct/spec.yaml

```diff
@@ -0,0 +1,25 @@
+$schema: https://azuremlschemas.azureedge.net/latest/model.schema.json
+name: tiiuae-falcon-40b-instruct
+path: ./
+properties:
+  SHA: ca78eac0ed45bf64445ff0687fabba1598daebf3
+  datasets: tiiuae/falcon-refinedweb
+  languages: en
+  evaluation-recommended-sku: Standard_NC24rs_v3
+  inference-recommended-sku: Standard_ND40rs_v2
+tags:
+  Featured: ''
+  Preview: ''
+  license: apache-2.0
+  task: text-generation
+  author: tiiuae
+  evaluation_compute_allow_list:
+  - Standard_NC24rs_v3
+  - Standard_ND40rs_v2
+  - Standard_ND96asr_v4
+  - Standard_ND96amsr_A100_v4
+  inference_compute_allow_list:
```

Comment:
inference on nc24 had failed?

Reply:
Deployment was successful but inference was not working on NC24 due to timeout.

Reply:
yes request timeout issue with NC24 compute
<img width="851" alt="image" src="https://github.com/Azure/azureml-assets/assets/38472520/133af47c-25d8-4397-a1be-dc5e9ff75474">